gcp/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -2,109 +2,45 @@ terraform {
   backend "gcs" {}
 }
 
-provider "google-beta" {
+provider "google" {
   project     = "${var.project_id}"
   credentials = "${var.serviceaccount_key}"
 }
 
-variable "nonce" {}
-variable "domain_name" {}
-variable "env" {}
-variable "organization_id" {}
-variable "project_id" {}
-variable "serviceaccount_key" {}
-variable "auth_user_email" {}
-
-# Terragrunt variables
-
-variable "notification_email" {}
-
-variable "use_auth_user_email" {
-  default = false
-}
-
-# Enables debug mode when TF_VAR_stackdriver_debug is not empty
-
-variable "stackdriver_debug" {
-  default = ""
+provider "google-beta" {
+  project     = "${var.project_id}"
+  credentials = "${var.serviceaccount_key}"
 }
 
-resource "template_dir" "resources" {
-  source_dir      = "${path.cwd}/resources"
-  destination_dir = "${path.cwd}/resources_rendered"
-
-  vars {
-    project_id         = "${var.project_id}"
-    organization_id    = "${var.organization_id}"
-    domain_name        = "${var.domain_name}"
-    notification_email = "${(var.use_auth_user_email && var.auth_user_email != "") ? var.auth_user_email : var.notification_email}"
-
-    enabled = "${(var.env == "prd" || var.env == "stg") ? true : false}"
-  }
+variable "crm_sa" {
+  default = "one-platform-tenant-manager@system.gserviceaccount.com"
```

Comment:
What is this service account?  Where does it come from?

Reply:
it seems to be the Cloud Resource Manager system account. CRM uses this SA to perform some operations on the resources of a project.

Reply:
@JC1738 CRM system account can modify IAM policies of the projects on behalf of certain APIs. It is needed here, so we can exclude IAM modification events that belong to CRM account from LBM that is used by IAM modification alert policy.